[2025-02-10T09:25:12.651+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T09:25:12.662+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cluster_deployment_versioned.download_stack_v1.0.0 manual__2025-02-10T09:25:01.240093+00:00 [queued]>
[2025-02-10T09:25:12.667+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cluster_deployment_versioned.download_stack_v1.0.0 manual__2025-02-10T09:25:01.240093+00:00 [queued]>
[2025-02-10T09:25:12.667+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T09:25:12.673+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonVirtualenvOperator): download_stack_v1.0.0> on 2025-02-10 09:25:01.240093+00:00
[2025-02-10T09:25:12.677+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=14465) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T09:25:12.678+0000] {standard_task_runner.py:72} INFO - Started process 14466 to run task
[2025-02-10T09:25:12.679+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cluster_deployment_versioned', 'download_stack_v1.0.0', 'manual__2025-02-10T09:25:01.240093+00:00', '--job-id', '529', '--raw', '--subdir', 'DAGS_FOLDER/cluster_deployment_versioned.py', '--cfg-path', '/tmp/tmpmg7mehrx']
[2025-02-10T09:25:12.680+0000] {standard_task_runner.py:105} INFO - Job 529: Subtask download_stack_v1.0.0
[2025-02-10T09:25:12.714+0000] {task_command.py:467} INFO - Running <TaskInstance: cluster_deployment_versioned.download_stack_v1.0.0 manual__2025-02-10T09:25:01.240093+00:00 [running]> on host 1999c8c59930
[2025-02-10T09:25:12.762+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cluster_deployment_versioned' AIRFLOW_CTX_TASK_ID='download_stack_v1.0.0' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T09:25:01.240093+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-10T09:25:01.240093+00:00'
[2025-02-10T09:25:12.763+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T09:25:12.763+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T09:25:12.763+0000] {logging_mixin.py:190} INFO - Current task name:download_stack_v1.0.0 state:running start_date:2025-02-10 09:25:12.662516+00:00
[2025-02-10T09:25:12.763+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_versioned and current dag run status:running
[2025-02-10T09:25:12.764+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T09:25:12.764+0000] {baseoperator.py:421} WARNING - PythonVirtualenvOperator.execute cannot be called outside TaskInstance!
[2025-02-10T09:25:12.765+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venv03wv3lgj --system-site-packages --python=python
[2025-02-10T09:25:12.766+0000] {process_utils.py:190} INFO - Output:
[2025-02-10T09:25:13.479+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.8.final.0-64 in 251ms
[2025-02-10T09:25:13.487+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venv03wv3lgj, clear=False, no_vcs_ignore=False, global=True)
[2025-02-10T09:25:13.488+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-02-10T09:25:13.489+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2025-02-10T09:25:13.493+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-02-10T09:25:13.501+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venv03wv3lgj/bin/pip install -r /tmp/venv03wv3lgj/requirements.txt
[2025-02-10T09:25:13.503+0000] {process_utils.py:190} INFO - Output:
[2025-02-10T09:25:14.722+0000] {process_utils.py:194} INFO - Requirement already satisfied: pandas==2.1.4 in /home/***/.local/lib/python3.12/site-packages (from -r /tmp/venv03wv3lgj/requirements.txt (line 1)) (2.1.4)
[2025-02-10T09:25:14.727+0000] {process_utils.py:194} INFO - Requirement already satisfied: numpy<2,>=1.26.0 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venv03wv3lgj/requirements.txt (line 1)) (1.26.4)
[2025-02-10T09:25:14.728+0000] {process_utils.py:194} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venv03wv3lgj/requirements.txt (line 1)) (2.9.0.post0)
[2025-02-10T09:25:14.729+0000] {process_utils.py:194} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venv03wv3lgj/requirements.txt (line 1)) (2024.2)
[2025-02-10T09:25:14.730+0000] {process_utils.py:194} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.12/site-packages (from pandas==2.1.4->-r /tmp/venv03wv3lgj/requirements.txt (line 1)) (2024.2)
[2025-02-10T09:25:14.731+0000] {process_utils.py:194} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.12/site-packages (from python-dateutil>=2.8.2->pandas==2.1.4->-r /tmp/venv03wv3lgj/requirements.txt (line 1)) (1.17.0)
[2025-02-10T09:25:15.253+0000] {process_utils.py:194} INFO - 
[2025-02-10T09:25:15.254+0000] {process_utils.py:194} INFO - [notice] A new release of pip is available: 24.3.1 -> 25.0.1
[2025-02-10T09:25:15.254+0000] {process_utils.py:194} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-02-10T09:25:15.356+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venv03wv3lgj/bin/python /tmp/venv-callg1ejaso3/script.py /tmp/venv-callg1ejaso3/script.in /tmp/venv-callg1ejaso3/script.out /tmp/venv-callg1ejaso3/string_args.txt /tmp/venv-callg1ejaso3/termination.log
[2025-02-10T09:25:15.357+0000] {process_utils.py:190} INFO - Output:
[2025-02-10T09:25:18.749+0000] {process_utils.py:194} INFO - [2025-02-10T09:25:18.748+0000] {script.py:30} INFO - Starting download stack...
[2025-02-10T09:25:28.767+0000] {process_utils.py:194} INFO - [2025-02-10T09:25:28.763+0000] {script.py:46} INFO - Writing download report to to: /opt/***/data/download_report_from_v1.0.0.json
[2025-02-10T09:25:28.773+0000] {process_utils.py:194} INFO - [2025-02-10T09:25:28.770+0000] {script.py:50} INFO - Download stack completed:
[2025-02-10T09:25:28.774+0000] {process_utils.py:194} INFO - Total size: 1200.0MB
[2025-02-10T09:25:28.775+0000] {process_utils.py:194} INFO - Total time: 240s
[2025-02-10T09:25:28.775+0000] {process_utils.py:194} INFO - Download speed: 5.0MB/s
[2025-02-10T09:25:29.608+0000] {python.py:240} INFO - Done. Returned value was: {'stacks': ['baseos', 'gi'], 'total_size_mb': 1200.0, 'total_time_seconds': 240, 'download_speed_mbs': 5.0, 'status': 'success', 'details': [{'stack': 'baseos', 'size_mb': 1000, 'status': 'completed', 'time': 180}, {'stack': 'gi', 'size_mb': 200, 'status': 'completed', 'time': 60}], 'timestamp': '2025-02-10T09:25:28.763082'}
[2025-02-10T09:25:29.658+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T09:25:29.658+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cluster_deployment_versioned, task_id=download_stack_v1.0.0, run_id=manual__2025-02-10T09:25:01.240093+00:00, execution_date=20250210T092501, start_date=20250210T092512, end_date=20250210T092529
[2025-02-10T09:25:29.670+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T09:25:29.670+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T09:25:29.671+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_versioned queued_at:2025-02-10 09:25:01.282535+00:00
[2025-02-10T09:25:29.671+0000] {logging_mixin.py:190} INFO - Task hostname:1999c8c59930 operator:PythonVirtualenvOperator
[2025-02-10T09:25:29.729+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T09:25:29.787+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T09:25:29.792+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
